glib/src/param_spec.rs

```diff
@@ -2174,6 +2174,35 @@ pub trait HasParamSpec {
     fn param_spec_builder() -> Self::BuilderFn;
 }
 
+// unless a custom `default` attribute is specified, the macro will use this trait.
+pub trait HasParamSpecDefaulted: HasParamSpec + Default {
+    type BuilderFnDefaulted;
+    fn param_spec_builder_defaulted() -> Self::BuilderFnDefaulted;
+}
+
+// Manually implement the trait for every Enum
+impl<
+        T: HasParamSpec<ParamSpec = ParamSpecEnum>
+            + StaticType
+            + FromGlib<i32>
+            + IntoGlib<GlibType = i32>
+            + Default,
+    > HasParamSpecDefaulted for T
+{
+    type BuilderFnDefaulted = fn(name: &str) -> ParamSpecEnumBuilder<T>;
+    fn param_spec_builder_defaulted() -> Self::BuilderFnDefaulted {
+        |name| Self::ParamSpec::builder(name)
+    }
+}
+
+// Manually implement the trait for chars
+impl HasParamSpecDefaulted for char {
```

Comment:
But for chars you can also just not provide `default` and it will default to 0 or so inside GLib